connectors/src/connectors/confluence/index.ts

```diff
@@ -220,6 +220,38 @@ export async function resumeConfluenceConnector(
   }
 }
 
+export async function pauseConfluenceConnector(
+  connectorId: ModelId
+): Promise<Result<undefined, Error>> {
+  const connector = await ConnectorResource.fetchById(connectorId);
+  if (!connector) {
+    logger.error({ connectorId }, "Connector not found.");
+    return new Err(new Error("Connector not found"));
+  }
+
+  await connector.markAsPaused();
```

Comment:
We should kill the running cron workflow, right?

Reply:
done

Reply:
👌